test/string.c

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright © 2017-2018 The Crust Firmware Authors.
+ * SPDX-License-Identifier: (BSD-3-Clause OR GPL-2.0)
+ */
+
+#include <stddef.h>
+#include <string.h>
+#include <unity.h>
+
+static void
+test_StrcmpReturnsCorrectAnswer(void)
+{
```

Comment:
We need test cases that cover cases where the string is _NOT_ the same (e.g. a return value of something other than 0).